code/game/objects/items/ego_weapons/non_abnormality/miscfixer.dm

```diff
@@ -1,8 +1,7 @@
 //Wedge - Grade 4 with two range.
 /obj/item/ego_weapon/city/wedge
 	name = "wedge office spear"
-	desc = "A black, ornate spear used by the wedge office."
-	icon_state = "wedge"
```

Comment:
Accidentally deleted the sprite.

Reply:
oh geez, whoops. i'll put that back in right away